app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -201,6 +208,18 @@ function EndpointController(
       endpoint.Type !== PortainerEndpointTypes.AgentOnKubernetesEnvironment
     ) {
       payload.URL = 'tcp://' + endpoint.URL;
+
+      if (endpoint.Type === PortainerEndpointTypes.DockerEnvironment) {
+        var tlsConfig = $scope.formValues.tlsConfig;
```

Comment:
isn't this the same as `getTLSValues` in APIForm?

Reply:
Do you mean replace `var tlsConfig = $scope.formValues.tlsConfig;` with `var tlsConfig = getTlsValues()`?

Reply:
If so, I don't see any added value with `getTlsValue()`;

Reply:
just to use the shared logic, instead of running it twice